controls/V-93285.rb

```diff
@@ -14,9 +14,9 @@
     Value Name: MaximumPasswordAge
 
     Value Type: REG_DWORD
-    Value: 0x0000001e (30) (or less, but not 0)"
-  desc  "fix", "This is the default configuration for this setting (30 days).
-    Configure the policy value for Computer Configuration >> Windows Settings >> Security Settings >> Local Policies >> Security Options >> \"Domain member: Maximum machine account password age\" to \"30\" or less (excluding \"0\", which is unacceptable)."
+    Value: 0x0000001e (#{input('maximum_password_age_machine')}) (or less, but not 0)"
```

Comment:
> Parameterized V-93285 based on #{input('maximum_password_age_machine')} [1c04a91](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/commit/1c04a91a48b242c8cc9b65127244885c9d1e46b6)

Parameterized some values that were still 30 https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/86437f3f6d97a370d96ce8cfc573eeecb07b3f1f
However, the hex value `0x0000001e` in the check text might be tricky to parameterize. I think we have a few options:
1. Remove the hex value altogether
2. Hardcode the hex value to `0x0000003c` (60) in the overlay as that is the only possible value for ARS HVA
3. Use Ruby to automatically convert integers (like 30 or 60) to hex by using: `30.to_s(16)`

If we use option 3, the code would look something like this:
```
Value Type: REG_DWORD
Value: 0x000000#{input('maximum_password_age_machine').to_s(16)} (#{input('maximum_password_age_machine')}) (or less, but not 0)
```

Reply:
Let's go ahead and try option 3. Thanks!

Reply:
Implemented option 3 https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/9acc401409129867cc0fffb0531dd30c90febfce